netci.groovy

```diff
@@ -68,7 +71,8 @@ def static getBuildJobName(def configuration, def os) {
             // TODO: set to false after tests are fully enabled
             def skipIfNoTestFiles = true
 
-            Utilities.setMachineAffinity(newJob, os, os == 'Windows_NT' ? 'latest-dev15' : 'latest-or-auto')
+            def affinity = configuration == 'Release_net40_no_vs' ? 'latest-or-auto' : (os == 'Windows_NT' ? 'latest-dev15' : 'latest-or-auto')
```

Comment:
I have no idea where to find out what gets installed with ``latest-or-auto`` or ``latest-dev15`` etc., or what the latest allowable values are   I've searched around for ``Utilities.setMachineAffinity`` but can't find the source for those utilites